src/edu/psu/sweng500/team8/play/CellChangedListener.java

```diff
@@ -3,6 +3,7 @@
 import edu.psu.sweng500.team8.coreDataStructures.Cell;
 
 /** Listener interface for cell number changes */
+/* TODO Isn't this creating circular dependency? */
```

Comment:
Indirectly, yes, but it's through an interface so it's not a big deal. Do you have a better suggestion for implementing events?